samples/Samples.h

```diff
@@ -40,6 +40,12 @@ extern "C" {
 #define SAMPLE_HASH_TABLE_BUCKET_COUNT  50
 #define SAMPLE_HASH_TABLE_BUCKET_LENGTH 2
 
+#define IOT_CORE_CREDENTIAL_ENDPOINT ((PCHAR) "AWS_IOT_CORE_CREDENTIAL_ENDPOINT")
```

Comment:
These are generic enough to actually go to a public header in the common sometimes in the future

https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/blob/master/src/include/com/amazonaws/kinesis/video/common/Include.h

Reply:
I noticed this thing, but I am not sure which way the team prefers. So I added it at the sample.h first. I noticed the team modifying this sample, so I left it to them. Good to see you. :)